torch/_dynamo/utils.py

```diff
@@ -2137,7 +2139,8 @@ def chromium_event_timed(
         log_pt2_compile_event,
     )
     try:
-        yield
+        with compile_time_record_function(f"{event_name} (chromium_event_timed)"):
```

Comment:
Err, should we move this before with an `enter` and then an `exit` in the finally?   This just matches the other events better imo